vertx-pojo-mapper-common/src/main/java/de/braintags/vertx/jomnigate/init/ObserverMapperSettings.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * #%L
+ * vertx-pojo-mapper-common
+ * %%
+ * Copyright (C) 2017 Braintags GmbH
+ * %%
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * #L%
+ */
+package de.braintags.vertx.jomnigate.init;
+
+import java.lang.annotation.Annotation;
+
+import de.braintags.vertx.jomnigate.exception.MappingException;
+import de.braintags.vertx.jomnigate.mapping.IMapper;
+
+/**
+ * The part of {@link ObserverSettings} which defines, on which {@link IMapper} the ObserverSettings shall be executed.
+ * The property classDefinition and annotation contain the definition. Examples for a class definition are:
+ * <UL>
+ * <LI>my.mapper.class<br/>
+ * the parent ObserverSettings will be applied for a mapper with the class my.mapper.class
+ * <LI>instanceof my.mapper.class<br/>
+ * the parent ObserverSettings will be applied for all classes, which are instanceof my.mapper.class
+ * </UL>
+ * 
+ * @author Michael Remme
+ * 
+ */
+public class ObserverMapperSettings {
+  private static final String INSTANCEOF = "instanceof";
+  private String classDefinition;
+  private boolean instOf = false;
+  private Class<?> mapperClass;
+  private Class<? extends Annotation> annotation;
+
+  @SuppressWarnings("unused")
+  private ObserverMapperSettings() {
+    // only usable for serialization
+  }
+
+  /**
+   * Constructor for a new instance. The class definition can be something like "my.mapper.class" or "instanceof
+   * my.mapper.class"
+   * 
+   * @param classDefinition
+   */
+  public ObserverMapperSettings(String classDefinition) {
+    setClassDefinition(classDefinition);
+  }
+
+  /**
+   * Constructor for a new instance, which shall be executed on mapper classes, which are annotated with the given
+   * annotation
+   * 
+   * @param annotation
+   */
+  public ObserverMapperSettings(Class<? extends Annotation> annotation) {
+    setAnnotation(annotation);
+  }
+
+  private void init() {
+    if (classDefinition != null) {
+      String clsName = classDefinition;
+      int index = clsName.indexOf(INSTANCEOF);
+      if (index >= 0) {
+        instOf = true;
+        clsName = clsName.substring(index + INSTANCEOF.length()).trim();
+      }
```

Comment:
But a configuration via magic strings is error prone and unclear to anyone not knowing the source code. 
I would never guess to add an "instanceof" in front of my class name, but if i see an "instanceof" field i know that i can assign a value to it.
The future addition of other configuration even speaks more to my solution, as otherwise the single string would get even longer, more complicated,and maybe even error prone if the order becomes important further on.
You already have all the fields needed for the configuration in the class itself, to take the direct JSON representation of the class for the configuration as JSON seems only logical.